### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --hidden --format brief`

### CODE REVIEW

File: tests/test_launcher.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def test_cleanup_not_immediate_per_test(
         LEAP_WITH_BIND_MOUNT_AND_VOLUME,
     ],
 )
-def test_launcher_creates_and_cleanes_up_volumes(
+def test_launcher_creates_and_cleanse_up_volumes(
```
Comment (dcermak) [inline tests/test_launcher.py:85]: This is not a typo, there is no cleansing going on

File: tests/test_launcher.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def test_launcher_creates_and_cleanes_up_volumes(
 
 
 @pytest.mark.parametrize("cont", [LEAP_WITH_VOLUME_IN_DOCKERFILE])
-def test_launcher_cleanes_up_volumes_from_image(
+def test_launcher_cleanse_up_volumes_from_image(
```
Comment (dcermak) [inline tests/test_launcher.py:126]: same here

**Review by dcermak — CHANGES_REQUESTED** [review]

**Comment by dcermak** [comment]
Thanks for the PR and catching the typos! I just flagged two issues that are not typos

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**